plugins/woocommerce/src/Internal/Orders/OrderStatusRestController.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
```

Comment:
```suggestion
<?php
declare( strict_types = 1 );

```

Required by phpcs linter rule.